scout/static/scout/js/filter.js

```diff
@@ -197,3 +197,9 @@ var Filter = {
         }
     }
 };
+
+/* node.js exports */
+if (typeof exports == "undefined") {
```

Comment:
We're still working on guidelines, but the pull request can't be so small that it doesn't solve a specific and current problem.  The problem this pull request solves doesn't exist yet, so the changes should be part of a pull request that does test these files.